esti/copy_test.go

```diff
@@ -0,0 +1,195 @@
+package esti
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/go-test/deep"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/require"
+	"github.com/treeverse/lakefs/pkg/api"
+	"github.com/treeverse/lakefs/pkg/block"
+	"github.com/treeverse/lakefs/pkg/block/azure"
+	"github.com/treeverse/lakefs/pkg/config"
+)
+
+const (
+	s3CopyDataPath            = "s3://esti-system-testing-data/copy-test-data/"
+	gsCopyDataPath            = "gs://esti-system-testing-data/copy-test-data/"
+	azureCopyDataPathTemplate = "https://%s.blob.core.windows.net/esti-system-testing-data/copy-test-data/"
+	azureAbortAccount         = "esti4multipleaccounts"
+	ingestionBranch           = "test-data"
+	largeObject               = "squash.tar"
+)
+
+var azureCopyDataPath = ""
```

Comment:
Not a fan of global variables - if both tests uses the helper function, they can pass the value. alternative, use helper function to return the copy data path.